client/actions/graphql/categoryGQLActions.js

```diff
@@ -8,15 +8,22 @@ import { GET_CLUBS, GET_CLUB, JOIN_CLUB, JOINED_CLUBS, UNJOIN_CLUB } from '../co
 import { handleError } from '../../utils/errorHandler';
 import Client from '../../client';
 
-
 export const getClubs = socialClubs => ({
   type: GET_CLUBS,
   payload: socialClubs,
   error: false,
 });
 
 export const getCategoryList = ({
-  before = '', after = '', first = 1, last = 1, name = '', name_Icontains = '', name_Istartswith = '', description_Icontains = '', description_Istartswith = ''
+  before = '',
+  after = '',
+  first = 1,
+  last = 1,
+  name = '',
+  name_Icontains = '',
```

Comment:
Curious, should n't this be camel case.

Reply:
@kenneth-kip the changes I made to this file was basically linting. I think fixing this should need a separate chore as it might be out of scope of the current PR's task.

Reply:
Yes you need make your `linting` not to be auto

Reply:
Okay @chibujax I will disable that.